index.html

```diff
@@ -1,10 +1,167 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Title</title>
+    <meta charset="UTF-8">
+    <title>Business Web-Opdracht</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<!--Header-->
+<header class="outer-container-purple">
+    <nav>
+        <ul>
+            <li><a href="#">Case Studies</a></li>
+            <li><a href="#">Services</a></li>
+            <li><a href="#">Contact US</a></li>
+        </ul>
+    </nav>
+
+    <div class="inner-container-short">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p class="small-weight-p">We are here to help you succeed.</p>
+        <button>Let's work together</button>
+        <img src="assets/squiggle_line_stripe.svg" alt="squiggle line" >
+    </div>
+
+    <div class="seprator"></div>
+
+</header>
+
+<!--main-->
+
+
+<main>
+
+<section class="section-intro"> <!--section intro-->
+    <p id="bg-hello">hello</p>
+    <div class="inner-container-short">
+        <h2>Rekupe is a digital agency in Los Angeles.</h2>
+        <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow. </h2>
+        <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+        <p>Have a question? <a href="#">Click here</a> to contact us!</p>
+    </div>
+
+</section>
+    <div class="seprator" id="grey-background"></div>
+
+<section class="section-info-blocks"><!--section info-blocks-->
+    <p id="bg-work">work</p>
+    <div class="inner-container-long">
+        <article class="info-block"> <!--info-block Rolling stone-->
+            <div>
+                <h2>Rolling Stone</h2>
+                <img src="assets/squiggle_line.svg" alt="squiggle line"/>
+                <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+                <a>VIEW PROJECT</a>
```

Comment:
Hardcoded capital letters doen we liever niet: stel dat iemand ineens van gedachte veranderd, dan moet jij dat overal gaan overtypen. Je kunt dit beter met CSS transformeren!